.az/mshost.yaml

```diff
@@ -424,6 +424,20 @@ jobs:
           ./gradlew --no-daemon clean connectedDebugAndroidTest
         displayName: Run extensions unit test with Android Emulator
         workingDirectory: $(Build.sourcesdirectory)
+           
+      - script: |
+          set -e -x
+          cd out/android/Release/java/androidtest/android
+          appcenter test run espresso \
+          --app "AI-Frameworks/Ort-Ext-Mobile-Android-Testapp" \
+          --devices "AI-Frameworks/android-pixel-5-galaxy-m10" \
+          --app-path ./app/build/outputs/apk/debug/app-debug.apk \
+          --test-series "master" \
+          --locale "en_US" \
+          --build-dir ./app/build/outputs/apk/androidTest/debug \
+          --token $(app_center_api_token_2)
```

Comment:
why do we have 2 API tokens?

Reply:
This is because I created the app with a newly generated token. 

I tried the original token it won't connect to the app center test run, so I linked another api token here.